src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -197,6 +202,77 @@ public decimal GetBalance(string accountName, int minConfirmations = 0)
             return balance?.ToUnit(MoneyUnit.BTC) ?? 0;
         }
 
+        [ActionName("listsinceblock")]
+        [ActionDescription("Get all transactions in blocks since block 'blockhash', or all transactions if omitted.")]
+        public async Task<ListSinceBlockModel> ListSinceBlockAsync(string blockHash, int targetConfirmations = 1)
+        {
```

Comment:
Did you base the logic on bitcoin core by any chance?

Reply:
This should be the same logic that is used in bitcoin core